Contributing.md

```diff
@@ -242,6 +242,14 @@ these guidelines:
  * Be sure your author field in git is properly filled out with your full name
  and email address so we can credit you.
 
+#### Need help with your pull request?
+
+Meteor now has groups defined to cover areas of the codebase. If you need help on certain topics you can address:
+
+* Meteor Data Team - This includes DDP, tracker, mongo, accounts, etc. You can talk to @stubailo.
+* Blaze - This includes Spacebars, Blaze, etc. You can talk to @urigo or @yyx990803.
+* Build tools - This includes modules, build tool changes, etc. You can talk to @avital or @benjamn.
```

Comment:
Maybe make links to their GitHub pages?

And also maybe place "talk" with how exactly you should talk. Like, no private messages probably.